### ISSUE TITLE
nextcloud on QNAP 

### ISSUE THREAD

**Issue by ariss77** [nextcloud/android#5844]
### Steps to reproduce
1. Input url - nextcloud.lan45.keenetic.pro
2.  Input login and pass
3.  is trying to get in, and no

If I enter an incorrect password - log in nextcloud I see a record of the attempt to login with an incorrect password

Enter through the browser on my phone - no problem

### Environment data
Android version: 9 

Device model: Xiaomi redmi note 5

Stock or customized system: stock

Nextcloud app version: latest

Nextcloud server version: 18.0.3

First there was a problem with the input through the browser via the domain name level 4 - complete description and solution of this problem here:
https://help.nextcloud.com/t/trusted-domains-i-do-not-know-what-to-add/77061/24

In short, how it works is through a domain keenetic.pro (supported by the router manufacturer)
"By default, remote access to the device on the domain level 4 through KeenDNS service works - I connect to the router using HTTPS, and then from the router goes online connection to the local device via the HTTP protocol."

Problem solved:
"Router support team suggested that nextcloud incorrectly handles the HTTP header, so I need to run the following commands (telnet command in the router):
ip http proxy nextcloud
preserve-host
exit
system configuration save
Did. And start working! "

Possibly with a mobile application because of this?

**Comment by tobiasKaminsky** [comment]
Nice that you found a solution :+1:

**Comment by ariss77** [comment]
The solution was found for the browser on the computer. 
For the android mobile app - no  :(((

**Comment by ariss77** [comment]
And you have already closed the topic.... :(
